web/modules/custom/block_module/src/Services/UserEmail.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Drupal\block_module\Services;
+
+/**
+ * Class to implement a custom service.
+ */
+class UserEmail {
+
+  /**
+   * Function  to return a string response.
+   *
+   * @return string
+   *   Returns a descriptive sentence.
+   */
+  public function result() {
+    $user = 'Hi, This response is from userEmail Service!';
+    return $user;
```

Comment:
Why decalare an extra variable?

Reply:
Isn't it better to have the message stored separately? Just in case later we need to change the message. Just in case I have a longer message to be returned or some logic involved.